### ISSUE TITLE
2018 Accessibility Trends

### ISSUE THREAD

**Issue by ljgoslin** [a11yproject/a11yproject.com#761]
2018 was a big year in Accessibility, join us in a recap of the major trends!

[This session](https://accessibility.deque.com/2018-accessibility-trends) is hosted by Deque's CEO, Preety Kumar, on **Thursday, April 25th** from **2:00-3:00PM EST**

Major tech and retail companies are showing up to (and sponsoring) accessibility conferences. More and more companies seem to have an interest in accessibility and in sustainable accessibility. From year to year, it is becoming abundantly clear that accessibility is growing and changing. 

This webinar is great for those who were unable to attend CSUN, missed this session at the show, or are curious about current accessibility trends. In this webinar attendees will learn about:

- Global accessibility trends for the US, Canadian, European, Chinese and Indian Markets
- Accessibility trends and growth for various verticals and market segments
- Accessibility technological trends and innovations

**Comment by ericwbailey** [comment]
Thanks! Closed via https://github.com/a11yproject/a11yproject.com/commit/9d72b0b97ad5b8287b9e26e33b92c086e95e2d39.
